internal/pkg/http/client.go

```diff
@@ -75,9 +68,14 @@ func (c Client) SendRequest(method, path string, headers []string, requestBody *
 
 		req.Header.Add(k, interpolatedHeaderValue)
 	}
-
+	if err != nil {
```

Comment:
Is this check necessary?

Reply:
Sorry, I thought we close the body in the previous check (line 56), too. This lgtm.